*** semantic-ctxt.el    07 Sep 2007 23:28:49 -0400      1.45
--- semantic-ctxt.el    15 Oct 2007 21:55:33 -0400
***************
*** 171,177 ****
  to collect tags, such as local variables or prototypes."
    ;; This assumes a bovine parser.  Make sure we don't do
    ;; anything in that case.
!   (when (and semantic--parse-table (not (eq semantic--parse-table t)))
      (let ((vars nil)
          (vars2 nil)
          ;; We want nothing to do with funny syntaxing while doing this.
--- 171,178 ----
  to collect tags, such as local variables or prototypes."
    ;; This assumes a bovine parser.  Make sure we don't do
    ;; anything in that case.
!   (when (and semantic--parse-table (not (eq semantic--parse-table t))
!            (not (semantic-parse-tree-unparseable-p)))
      (let ((vars nil)
          (vars2 nil)
          ;; We want nothing to do with funny syntaxing while doing this.
